src/core/text/Text.js

```diff
@@ -273,7 +273,10 @@ Text.prototype.updateText = function (respectDirty)
     if (navigator.isCocoonJS)
```

Comment:
I did a basic benchmark test, just doing a loop over calling clearrect, and even going upto a million clearRects, can not see a difference, on a canvas of 1920x1080, between a clearRect of 1x1 and a clearRect of the full 1920x1080. Tried on Firefox, IE and Chrome on the desktop, Cocoon on an android device with Canvas+.

So I've changed my mind and am now leaning towards going the cleaner route and clear the entire canvas all of the time at that point ie. lose the if statement of only doing it for cocoon.